galactic/release-build.yaml

```
@@ -15,6 +15,8 @@ notifications:
   - scott+build.ros2.org@openrobotics.org
   - ros2-buildfarm-galactic@googlegroups.com
   maintainers: true
+package_whitelist:
+- null
```

I expect this is required because if the entire field is null then it's processed as if there is no whitelist at all. Is an empty whitelist treated the same as a null / absent one? IMO that would be much clearer than a list with a single `null` entry which could play games with the python code trying to find a package whose name is the python value `None`.

```suggestion
package_whitelist: []
```